packages/edit-site/src/components/style-book/style.scss

```diff
@@ -18,22 +22,18 @@
 }
 
 .edit-site-style-book__tablist-container {
-	background: $white;
+	flex-shrink: 0;
```

Comment:
someone, I don't remember who right now, suggested using the flex shorthand in a different PR, very minor thing but maybe worth doing for consistency.

Reply:
Sure thing!

Personally, with flex I usually prefer the individual properties because seeing "grow" and "shrink" makes it much easier for a developer reading the code to predict the behavior. But consistency FTW!